src/main/java/org/jabref/gui/autocompleter/AutoCompletePreferences.java

```diff
@@ -13,15 +13,15 @@
     private AutoCompleteFirstNameMode firstNameMode;
     private boolean onlyCompleteLastFirst;
     private boolean onlyCompleteFirstLast;
-    private List<String> completeNames;
-    private JournalAbbreviationPreferences journalAbbreviationPreferences;
+    private List<String> completeFields;
+    private final JournalAbbreviationPreferences journalAbbreviationPreferences;
 
     public AutoCompletePreferences(boolean shouldAutoComplete, AutoCompleteFirstNameMode firstNameMode, boolean onlyCompleteLastFirst, boolean onlyCompleteFirstLast, List<String> completeNames, JournalAbbreviationPreferences journalAbbreviationPreferences) {
         this.shouldAutoComplete = shouldAutoComplete;
         this.firstNameMode = firstNameMode;
         this.onlyCompleteLastFirst = onlyCompleteLastFirst;
         this.onlyCompleteFirstLast = onlyCompleteFirstLast;
-        this.completeNames = completeNames;
+        this.completeFields = completeNames;
```

Comment:
Why not renaming the parameter, too? Maybe `fieldsToComplete` would be a good name for both variables? `completeFields` is also OK for me.